meta-resin-common/recipes-support/resin-mounts/resin-mounts/mnt-conf-systemd.automount

```diff
@@ -0,0 +1,8 @@
+[Unit]
+Description=Automount systemd presistent state
+
+[Automount]
```

Comment:
OK it would work but this would mean that this mount will be different then all the others. Can we unify this for now and then decide if this is an easier approach?